package.json

```diff
@@ -115,6 +115,7 @@
   },
   "devDependencies": {
     "@faker-js/faker": "^9.3.0",
+    "@google-cloud/bigquery": "^7.9.1",
```

Comment:
@flozia I see you added this dependency, which we got a [dependabot PR](https://github.com/mozilla/blurts-server/pull/5656) for. Could you add a section to [the docs](https://github.com/mozilla/blurts-server/blob/main/docs/dependency-updates.md#npm) describing how to check that the update was OK, i.e. what command to run and what results we should expect to see if successful?

Reply:
https://github.com/mozilla/blurts-server/pull/6109